### PR TITLE
Properly name and separate each steps at publication.

### PR DESCRIPTION
**Pull request by jerome-benoit**
Signed-off-by: Jérôme Benoit <jerome.benoit@sap.com>

## PR Checklist

- [x] Please add a description of your changes.
- [ ] We need your changes to come with unit tests in order to keep this project in quality and easy to maintain.
- [ ] Please add a link to the open issue or task that this pull request will resolve.
- [ ] Add a note to the changelog to indicate the change.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

If yes please indicate it and migration info into the CHANGELOG.md file.

---

### CODE REVIEW

File: .github/workflows/npmpublish.yml
Comment (pioardi) [inline .github/workflows/npmpublish.yml:54]: We know it is present, is not better to fail if is not ?

Comment (jerome-benoit) [reply .github/workflows/npmpublish.yml:54]: It's better to fail, I will remove the CLI options.

**Review by Shinigami92 — APPROVED** [review]

**Comment by jerome-benoit** [comment]
@pioardi it's trivial cleanup, so I merge it without waiting for more approval. I will of course not do the same for PR that are not trivial ;)

**Pull request merged**
